bigbluebutton-html5/imports/api/users/server/methods/userLeaving.js

```diff
@@ -30,11 +30,11 @@ export default function userLeaving(credentials, userId) {
       'user-not-found', `You need a valid user to be able to toggle audio`);
   }
 
-  if (User.user.connection_status === OFFLINE_CONNECTION_STATUS) {
+  if (User.connection_status === OFFLINE_CONNECTION_STATUS) {
```

Comment:
👍 
The exception on the User is probably when you join as a Flash user but dont worry about that I have a branch with the fix already, will probably send the PR this week.